llvm/projects/ton-compiler/cpp-sdk/tvm/dict.hpp

```diff
@@ -0,0 +1,171 @@
+#pragma once
+
+#include <tuple>
+
+#include <tvm/cell.hpp>
+#include <tvm/slice.hpp>
+#include <tvm/builder.hpp>
+
+#include <tvm/schema/make_builder.hpp>
+#include <tvm/schema/make_parser.hpp>
+#include <tvm/dictionary.hpp>
+
+namespace tvm {
+
+/**
+ * Just a wrapper class over `dictionary` class with key length as a template
+ * argument. This way we avoid passing the key length to each method.
+ * @tparam KeyLen size in bits of the key
+ */
+template<unsigned KeyLen>
+class Dict {
+public:
+  using key_type = uint_t<KeyLen>;
+
+  template<unsigned K>
```

Comment:
Can you elaborate on what's the purpose of this template parameter `<K>`? Isn't this the same as `KeyLen`?

Reply:
Yep, you right. Fixed, thanks.